tools/README.md

```diff
@@ -0,0 +1,20 @@
+# Tools
+
+## Rotating Certificates
+
+To rotate certificates for all CNV components, type:
+
+```
+export KUBECONFIG=<wherever>
+export KUBECTL_BINARY=kubectl
+./rotate-certs.sh -n kubevirt-hyperconverged
+```
+
+During the rotation, the following things may occure:
+
+ * Migrations will be cancelled
+ * Image uploads will be cancelled
+ * VNC and Console connections will be closed
+
+After the rotation is done, all opperations will continue as usual.
+VirtualMachine and VirtualMachineInstance workloads will not be affected.
```

Comment:
CDI import and clone operations will also be unaffected.

Reply:
Are you sure? @mhenriks asked me to delete all cdi pods in all namespaces.